redbot/cogs/downloader/downloader.py

```diff
@@ -1119,36 +1119,54 @@ async def _cog_checkforupdates(self, ctx: commands.Context) -> None:
         await self.send_pagified(ctx, message)
 
     @cog.command(name="update")
-    async def _cog_update(self, ctx: commands.Context, *cogs: InstalledCog) -> None:
+    async def _cog_update(
+        self, ctx: commands.Context, reload: Optional[bool], *cogs: InstalledCog
+    ) -> None:
         """Update all cogs, or ones of your choosing.
 
         Examples:
             - `[p]cog update`
+            - `[p]cog update True`
             - `[p]cog update defender`
+            - `[p]cog update True defender`
 
         **Arguments**
 
+        - `[reload]` Whether to reload or not
```

Comment:
Not a full review (since it's 1am), but this description is not clear enough and is missing a period.